install/kubernetes/cilium/templates/clustermesh-apiserver/deployment.yaml

```diff
@@ -263,19 +263,28 @@ spec:
         - --cluster-name=$(CLUSTER_NAME)
         - --cluster-id=$(CLUSTER_ID)
         - --kvstore-opt=etcd.config=/var/lib/cilium/etcd-config.yaml
+        - --kvstore-opt=etcd.bootstrapQps=10000
         - --kvstore-opt=etcd.qps=100
         - --kvstore-opt=etcd.maxInflight=10
         - --clustermesh-config=/var/lib/cilium/clustermesh
         {{- if hasKey .Values.clustermesh "maxConnectedClusters" }}
         - --max-connected-clusters={{ .Values.clustermesh.maxConnectedClusters }}
         {{- end }}
+        - --health-port={{ .Values.clustermesh.apiserver.kvstoremesh.healthPort }}
         {{- if .Values.clustermesh.apiserver.metrics.kvstoremesh.enabled }}
         - --prometheus-serve-addr=:{{ .Values.clustermesh.apiserver.metrics.kvstoremesh.port }}
         - --controller-group-metrics=all
         {{- end }}
         {{- with .Values.clustermesh.apiserver.kvstoremesh.extraArgs }}
         {{- toYaml . | trim | nindent 8 }}
         {{- end }}
+        readinessProbe:
+          httpGet:
+            path: /readyz
+            port: kvmesh-health
+          periodSeconds: 30
```

Comment:
We can probably reduce it or was there a specific reason for picking 30s?

Reply:
Tried to pick a useful but non-aggressive period here, since this will continue to run indefinitely even after the container is ready. Do you think a lower value would be better?

Reply:
ah good point, how about we use startupProbe instead?

Reply:
I considered using a startupProbe (for clustermesh-apiserver readiness as well), but it will restart the Pod on failure (like livenessProbe) instead of just keeping it out of rotation until its ready. It would be nice if there was a `startup` level probe for readiness as well. It would also be difficult to pick a value that would be appropriate for small and large scale clustermesh deployments

Reply:
Could be worth adding a startup probe now that we have a global circuit breaker, so that we can check more aggressively whether kvstoremesh turned ready.